app/src/main/AndroidManifest.xml

```diff
@@ -18,22 +18,20 @@
         android:networkSecurityConfig="@xml/network_security_config"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:theme="@style/AppTheme"
         tools:replace="android:name">
-
+        <!--android:theme="@style/BlackTheme" Placed above-->
```

Comment:
Do we need these comments on this file? If not, make sure to remove then later.

Reply:
Oh no we don't. These are just for me and I will be removing them later.